src/components/sequencing/form/utils.ts

```diff
@@ -122,6 +122,13 @@ export function unquoteUnescape(s: string) {
   return s;
 }
 
+export function removeQuotes(text: string | number | boolean): string | number | boolean {
```

Comment:
This file is redundant. I think it was left over from a previous PR. Essentially my feedback for this was to move to the `/utilities` directory. It currently is there as `codemirror-utils.ts`. Do you mind removing this file and making the change in the `codemirror-utils.ts` file instead?

Reply:
Also do you mind adding a test for at least this function? Bonus points for the rest of the untested functions.

Reply:
Ideally these tests are unit tests that test the immediate logic invoked so that it's more immediately obvious to see which portion of the code is broken via looking at the tests. The logic being tested in `from-seq-json` is much more than just `removeQuotes` and `quoteEscape`. Super ideally, all the util functions from `codemirror-utils` should be mocked to completely isolate the test to just `from-seq-json` and make it a "true" unit test, but I don't think that's super important right now.

Reply:
I see, no problem I just added a `codemirror-utils.test.ts` to the PR.